shared_rubocop.yml

```diff
@@ -154,3 +154,11 @@ Lint/RaiseException:
 # Skip redundant "begin"/"end" at the top level of method definitions or blocks.
 Style/RedundantBegin:
   Enabled: true
+
+# Warn about too many positional arguments in the parameter list when more than 5.
+# Warn about "too many positional arguments with optional values" when more than 3.
+Metrics/ParameterLists:
+  Enabled: true
+  CountKeywordArgs: false
+  Max: 5
+  MaxOptionalParameters: 3
```

Comment:
Out of curiosity, did you try setting this lower, and did that find anything in Auctionet core? I feel like we could set this to at least 2, possibly 1. But we could do that later, of course.

Reply:
I was unable to run it anywhere due to the solargraph gem holding rubocop back.